flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Nix flake for dae and daed";
+
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    pnpm2nix = {
+      url = "github:Ninlives/pnpm2nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+    pre-commit-hooks = {
+      url = "github:cachix/pre-commit-hooks.nix";
+      inputs.nixpkgs.follows = "nixpkgs";
+    };
+  };
+
+  outputs = inputs@{ self, flake-parts, pre-commit-hooks, nixpkgs, ... }:
+    flake-parts.lib.mkFlake { inherit inputs; } {
+      imports = [
+      ];
+      systems = [ "x86_64-linux" "aarch64-linux" ];
+      perSystem = { config, self', inputs', pkgs, system, ... }: {
+        packages =
+          let
+            version = pkgs.lib.substring 0 8 self.lastModifiedDate
```

Comment:
What's the expected version value? How is it fetched?

Reply:
> What's the expected version value? How is it fetched?

This depends on the last modified date of this repository. The version today is `20230727`.

Reply:
So you mean that this version is for `flake` not for `dae|daed`?

Reply:
> > What's the expected version value? How is it fetched?
> 
> This depends on the last modified date of this repository. The version today is `20230727`.

The head commit hash and source hash for both `dae` and `daed` will be automatically updated by the bot to the `dynamic_info` file. The version will be based on the submission date of the latest commit in either `dae` or `daed` repositories (assuming that whenever either of these repositories is committed, the hash in this repository is immediately updated as well). 

Alternatively, it could be changed to directly use a short rev hash, which would allow users to avoid rebuilding the other package when only one of them is updated. The downside is that this approach may be less intuitive.